examples/dotnet/README.md

```diff
@@ -0,0 +1,36 @@
+# Running all test from Selenium C# example
+
+Follow these steps to run all test examples from selenium C#
+
+1. Clone this repository
+
+```
+git clone https://github.com/SeleniumHQ/seleniumhq.github.io.git
+```
+
+2. Navigate to `dotnet` directory
+
+```
+cd seleniumhq.github.io/examples/dotnet
+```
+
+3. Install dependecies using dotnet
+
+```
+dotnet restore
```

Comment:
So, this doc doesn't say/explain on installing and using the `dotnet` command. Adding prerequisites on dotnet installation steps could be beneficial. Any thoughts ?

This comment applies for all other lang readme.md!

Reply:
That does seem like a good idea, however, would the prerequisite also need to include steps for installing on different OS's? For instance, one set of steps for MacOS, one set of steps for Windows.

Reply:
HI @sadik312 ,

Pointing to [this link on Microsoft Learn](https://learn.microsoft.com/en-us/dotnet/core/install/) should be sufficient, as it covers all cross-platform installation steps for .NET